## Урок 5

### Задача 1

Известно, что генеральная совокупность распределена нормально
со средним квадратическим отклонением, равным 16.
Найти доверительный интервал для оценки математического ожидания a с надежностью 0.95,
если выборочная средняя M = 80, а объем выборки n = 256.

Доверительный интервал считаем по формулам: $ X - Z_{a/2} * \frac{Std}{\sqrt{n}}$ и $ X + Z_{a/2} * \frac{Std}{\sqrt{n}}$

In [7]:
80 - 1.96 * 16/(256**(1/2))

78.04

In [6]:
80 + 1.96 * 16/(256**(1/2))

81.96

[78.04, 81.96]

### Задача 2

В результате 10 независимых измерений некоторой величины X, выполненных с одинаковой точностью,
получены опытные данные:
6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1
Предполагая, что результаты измерений подчинены нормальному закону распределения вероятностей,
оценить истинное значение величины X при помощи доверительного интервала, покрывающего это
значение с доверительной вероятностью 0,95.

Поскольку мы не знаем std гс, то доверительный интервал будем считать по формулам: $ X - t_{a/2} * \frac{Std}{\sqrt{n}}$ и $ X + t_{a/2} * \frac{Std}{\sqrt{n}}$

In [9]:
import numpy as np

In [12]:
x = np.array([6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1])
x

array([6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1])

In [21]:
X = x.mean()
X

6.590000000000001

In [22]:
std = x.std(ddof=1)
std

0.4508017549014448

In [26]:
X - 2.262 * std / (10 ** (1/2))

6.267538255912426

In [25]:
X + 2.262 * std / (10 ** (1/2))

6.912461744087575

[6.267, 6.921]

### Задача 3

Утверждается, что шарики для подшипников, изготовленные автоматическим станком, имеют средний диаметр 17 мм.
Используя односторонний критерий с α=0,05, проверить эту гипотезу, если в выборке из n=100 шариков средний диаметр
оказался равным 17.5 мм, а дисперсия известна и равна 4 кв. мм.

$H_0: mu = mu_0$ <br>
$H_1: mu > mu_0$  

$Z_t = 1.96$

$Z_H = \frac{mu - mu_0}{\frac{std}{\sqrt{n}}}$

In [31]:
(17.5 - 17) / (4**(1/2) / 100**(1/2))

2.5

Верна гипотеза H1, при а=5% 

### Задача 4

Продавец утверждает, что средний вес пачки печенья составляет 200 г.
Из партии извлечена выборка из 10 пачек. Вес каждой пачки составляет:
202, 203, 199, 197, 195, 201, 200, 204, 194, 190.
Известно, что их веса распределены нормально.
Верно ли утверждение продавца, если учитывать, что доверительная вероятность равна 99%?

In [33]:
x = np.array([202, 203, 199, 197, 195, 201, 200, 204, 194, 190])
x

array([202, 203, 199, 197, 195, 201, 200, 204, 194, 190])

In [34]:
mu1 = x.mean()
mu1

198.5

In [36]:
std = x.std(ddof=1)
std

4.453463071962462

$T_H = \frac{mu - mu_0}{\frac{std}{\sqrt{n}}}$

In [38]:
(mu1 - 200) / (std / 10 ** (1/2))

-1.0651074037450896

Tt = 3.25

Верна гипотеза H0, при а=1%